### PR TITLE
[MOSIP-37256 ] adding readme

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,28 @@
...
+
+1. Build and install:
+   $ cd commons-packet <br>$ mvn clean install -Dgpg.skip=true
+___
+## Build Docker for a service
+>$ cd `<service folder>` <br>
+$ docker build -f Dockerfile
+# Property file used by the application
+We are using two property files that are accessible in this [repository](https://github.com/mosip/mosip-config/blob/develop/).
+> 1: [application-default.properties](https://github.com/mosip/mosip-config/blob/develop/application-default.properties)<br> 2:[ packet-manager-default.properties](https://github.com/mosip/mosip-config/blob/develop/packet-manager-default.properties)
```
Comment (ashok-ksharma) [inline README.md:25]: We are using below configuration files which are accessible from this [repository](<Add link to master branch>). Please refer the required tagged version for configuration.
application-default.properties <Add the link to the master>
 packet-manager-default.properties <Add the link to the master>

File: README.md [hunk 2]
```diff
@@ -6,4 +6,28 @@
...
+## Build & run (for developers)
+The project requires JDK 21.0.3  and mvn version 3.9.6
+
+1. Build and install:
+   $ cd commons-packet <br>$ mvn clean install -Dgpg.skip=true
+___
+## Build Docker for a service
+>$ cd `<service folder>` <br>
+$ docker build -f Dockerfile
+# Property file used by the application
```
Comment (ashok-ksharma) [inline README.md:23]: Use this heading as "Configuration" as we are following the same at other places. Check the text size as well. It is larger than other texts. It should be same.

File: README.md [hunk 3]
```diff
@@ -6,4 +6,28 @@
 ## About
 * Its used by "Registration Client" and "Resident Service" to create packets.
 * Its used by "Registration Processor" to read packet.
-  
+
+## Overview
+
+* This repository contains the source code MOSIP Packet Manager module.  For an overview refer [here](https://docs.mosip.io/1.2.0/modules/packet-manager). This Module Exposes some API endpoints that we can explore in this [Swagger](https://api-internal.dev.mosip.net/commons/v1/packetmanager/swagger-ui/index.html?configUrl=/commons/v1/packetmanager/v3/api-docs/swagger-config#/).
```
[the remark this answers is not in the document]
Comment (Khuddusshariff0022) [reply README.md:12]: added as "https://<RESPECTIVE_ENVIRONMENT>/commons/v1/packetmanager/swagger-ui/index.html"

File: README.md [hunk 4]
```diff
@@ -6,4 +6,28 @@
 ## About
 * Its used by "Registration Client" and "Resident Service" to create packets.
 * Its used by "Registration Processor" to read packet.
-  
+
+## Overview
```
[the remark this answers is not in the document]
Comment (Khuddusshariff0022) [reply README.md:10]: Done

File: README.md [hunk 5]
```diff
@@ -6,4 +6,28 @@
...
+
+* This repository contains the source code MOSIP Packet Manager module.  For an overview refer [here](https://docs.mosip.io/1.2.0/modules/packet-manager). This Module Exposes some API endpoints that we can explore in this [Swagger](https://api-internal.dev.mosip.net/commons/v1/packetmanager/swagger-ui/index.html?configUrl=/commons/v1/packetmanager/v3/api-docs/swagger-config#/).  
+---
+## Build & run (for developers)
+The project requires JDK 21.0.3  and mvn version 3.9.6
+
+1. Build and install:
+   $ cd commons-packet <br>$ mvn clean install -Dgpg.skip=true
```
[the remark this answers is not in the document]
Comment (Khuddusshariff0022) [reply README.md:18]: Done.

File: README.md (hunk 1 above)
Comment (Khuddusshariff0022) [reply README.md:25]: Done

File: README.md (hunk 2 above)
Comment (Khuddusshariff0022) [reply README.md:23]: Done.

File: README.md [hunk 6]
```diff
@@ -3,7 +3,49 @@
...
+
+---
+
+## Test 
+Tests are performed for this repository using [DSL](https://github.com/mosip/mosip-automation-tests) automation testing.
+
+---
+
+## APIs
+API documentation is available [here](https://mosip.github.io/documentation/1.2.0/commons-packet-service.html).
```
Comment (ashok-ksharma) [inline README.md:46]: Can we add below links for API:
https://mosip.github.io/documentation/

**Review by ashok-ksharma — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (Khuddusshariff0022) [reply README.md]: Done

**Review by ashok-ksharma — APPROVED** [review]

**Pull request merged**